### PR TITLE
Fix CORS Error

### PR DESCRIPTION
**Pull request by 5ym**
The root cause is not known, but the presence of crossorigin causes a CORS error

### CODE REVIEW

**Comment by timlrx** [comment]
Thanks for the fix, have you verified that it works with the changes?

**Comment by 5ym** [comment]
Adding a tag without crossorigin in tailwind-nextjs-starter-blog works fine.
Screenshot 2023-08-28 at 9 24 02 AM

File: packages/pliny/src/comments/Disqus.tsx [hunk 1]
```diff
@@ -12,28 +13,25 @@ export type DisqusProps = DisqusConfig['disqusConfig'] & {
...
+      script.setAttribute('data-timestamp', Date.now().toString())
       script.async = true
       document.body.appendChild(script)
     } else {
-      //@ts-ignore
-      window.DISQUS.reset({ reload: true })
+      ;(window as any).DISQUS.reset({ reload: true })
     }
-  }, [shortname, slug])
+  }, [shortname, slug, nextTheme])
```
Comment (5ym) [inline packages/pliny/src/comments/Disqus.tsx:34]: Text will not be visible unless reloaded when the theme is switched

Comment (timlrx) [reply packages/pliny/src/comments/Disqus.tsx:34]: Thanks, I think no need to alias `theme` to `nextTheme`.

**Review by timlrx — APPROVED** [review]
Thanks for the fix!

**Comment by 5ym** [comment]
@timlrx 
Thank you.
Please let me know if you need any corrections.

**Comment by timlrx** [comment]
Could you add a patch changeset - `yarn changeset` and commit the changes? I will merge it in to trigger a release.

**Comment by 5ym** [comment]
@timlrx 
I added it. Is it correct?

**Comment by timlrx** [comment]
Yup, this is good. Thanks!

**Pull request merged**